core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2547,6 +2547,8 @@ object SparkContext extends Logging {
   private[spark] val SPARK_SCHEDULER_POOL = "spark.scheduler.pool"
   private[spark] val RDD_SCOPE_KEY = "spark.rdd.scope"
   private[spark] val RDD_SCOPE_NO_OVERRIDE_KEY = "spark.rdd.scope.noOverride"
+  private[spark] val IS_INDETERMINATE_STAGE = "spark.stage.isIndeterminateStage"
```

Comment:
why do we need this? If `_shuffle_generation_id` is not in the task context property, then it's not an indeterminate stage.

Reply:
It's only used to mark `taskSet` belongs to a retried indeterminate stage, as you comment in https://github.com/apache/spark/pull/24110#discussion_r272449374, if we agree to add the parameter in TaskSet, it can be removed.